### PR TITLE
Workaround postfix issues without IPv6

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Comment by lhellebr** [comment]
Passed: https://satellite-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/lhellebr-provisioning-6.7-rhel7/8/consoleFull

**Comment by lpramuk** [comment]
Spotted the issue during rpm install:
```
[root@sat.example.com] out: ================================================================================
[root@sat.example.com] out: Install    2 Packages
[root@sat.example.com] out: Upgrade  214 Packages
[root@sat.example.com] out: 
[root@sat.example.com] out: Total download size: 292 M
[root@sat.example.com] out: Delta RPMs disabled because /usr/bin/applydeltarpm not installed.

[root@sat.example.com] out: warning: /etc/os-release saved as /etc/os-release.rpmsave
[root@sat.example.com] out: newaliases: fatal: parameter inet_interfaces: no local interface found for ::1
[root@sat.example.com] out: warning: /etc/shadow created as /etc/shadow.rpmnew
```

**Comment by lpramuk** [comment]
postfix has issue since we disabled IPv6 and `lo` iface is lacking `::1` address

so the logic is opposite not issue `... with IPv6` but `... without IPv6`

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -3260,3 +3261,12 @@ def package_install(packages, sat_version=None):
     else:
         command = 'yum -y install {}'.format(packages)
     return command
+
+
+def setup_sendmail():
+    # there are some issues with mailing when using sendmail; workaround them:
+    # - with IPv6, mailing won't work => force postfix to use IPv4
```
Comment (lpramuk) [inline automation_tools/__init__.py:3268]: please rephrase the comment accordingly to my ltest findings: 
- without IPv6 address for lo interface mailing won't work

**Comment by lhellebr** [comment]
Required by https://github.com/SatelliteQE/robottelo/pull/7777

**Comment by lhellebr** [comment]
@SatelliteQE/quality-engineers  This needs one more ACK

**Comment by pondrejk** [comment]
@lhellebr ok with merge, but please resolve the conflict

**Comment by lhellebr** [comment]
@pondrejk Conflict resolved

**Pull request merged**